core/crypto/src/signature.rs

```diff
@@ -133,6 +133,13 @@ pub enum PublicKey {
 }
 
 impl PublicKey {
+    pub fn len(&self) -> usize {
+        match self {
+            PublicKey::ED25519(_) => ed25519_dalek::PUBLIC_KEY_LENGTH + 1,
+            PublicKey::SECP256K1(_) => 65,
```

Comment:
hmm is there some constant like `ed25519_dalek::PUBLIC_KEY_LENGTH` that we can use here?

Reply:
Seems not, I didn't find it, but we can make it

Reply:
It is literally what @bowenwang1996 said (`ed25519_dalek::PUBLIC_KEY_LENGTH`)

Reply:
@SkidanovAlex But we need the const for Secp256k1, we already have the cost for ed25519...